src/views/AddLead/index.js

```diff
@@ -464,26 +514,74 @@ class AddLead extends React.PureComponent {
                         label={projectInputLabel}
                         options={projects}
-                        keySelector={AddLead.projectKeySelector}
-                        labelSelector={AddLead.projectLabelSelector}
-                    />
-                    <TextInput
-                        faramElementName="title"
-                        label={titleInputLabel}
+                        keySelector={projectKeySelector}
+                        labelSelector={projectLabelSelector}
                     />
+                    <div className={_cs(styles.inputButtonGroup, styles.titleGroup)}>
+                        <TextInput
+                            className={styles.input}
+                            faramElementName="title"
+                            label={titleInputLabel}
+                        />
+                        <div className={styles.buttons}>
+                            <AccentButton
+                                className={styles.smallButton}
+                                title="Format"
+                                onClick={this.handleAutoFormatTitleButton}
+                            >
+                                Aa
+                            </AccentButton>
+                        </div>
+                    </div>
+                    {(
+                        (
+                            suggestedTitleFromUrl
+                            && (title !== suggestedTitleFromUrl)
+                        ) || (
+                            suggestedTitleFromExtraction
+                            && (title !== suggestedTitleFromExtraction)
+                        )
+                    ) && (
+                        <>
+                            <h5 className={styles.suggestionLabel}>
+                                Title suggestions:
+                            </h5>
+                            <div className={styles.suggestions}>
+                                {((title !== suggestedTitleFromUrl)
+                                    && (suggestedTitleFromUrl !== suggestedTitleFromExtraction)
+                                ) && (
+                                    <BadgeInput
+                                        className={styles.suggestionBadge}
+                                        faramElementName="title"
+                                        title={suggestedTitleFromUrl}
+                                    />
+                                )}
+                                {(title !== suggestedTitleFromExtraction) && (
+                                    <BadgeInput
+                                        className={styles.suggestionBadge}
+                                        faramElementName="title"
+                                        title={suggestedTitleFromExtraction}
+                                    />
+                                )}
+                            </div>
+                        </>
+                    )}
```

Comment:
This can be greatly simplified if we create a list unique(["suggestedTitleFromUrl", "suggestedTitleFromExtraction"]).filter(value => value !== "alreadySelectedValue")